client/app/components/datagrid/OperationDataGrid.tsx

```diff
@@ -42,7 +47,7 @@ const OperationDataGrid = ({
             {/* 🔗 Add reg or details link */}
             <Link
               className="no-underline text-bc-link-blue whitespace-normal"
-              href={`operations/${params.row.id}/1`}
+              href={`operations/${params.row.id}${isIndustryUser ? "/1" : ""}`}
```

Comment:
This is unrelated to this PR though I noticed it was broken. I'm unsure of what happened here as this was previously implemented. Likely a rebase issue.